lua/terrortown/entities/items/item_ttt_armor.lua

```diff
@@ -36,4 +36,75 @@ else -- CLIENT
 		-- limit item when in classic mode
 		item.limited = true
 	end)
+
+	---
+	-- @ignore
+	function ITEM:AddToSettingsMenu(parent)
+		local form = vgui.CreateTTT2Form(parent, "header_equipment_additional")
+
+		form:MakeHelp({
+			label = "help_armor_balancing"
+		})
+
+		form:MakeCheckBox({
+			serverConvar = "ttt_item_armor_block_headshots",
+			label = "label_armor_block_headshots"
+		})
+
+		form:MakeCheckBox({
+			serverConvar = "ttt_item_armor_block_blastdmg",
+			label = "label_armor_block_blastdmg"
+		})
+
+		form:MakeHelp({
+			label = "help_item_armor_classic"
+		})
+
+		form:MakeCheckBox({
+			serverConvar = "ttt_armor_classic",
+			label = "label_armor_classic"
+		})
```

Comment:
I think we should only put settings in here that are directly related to the specific item and not the armor system in general.

Reply:
I mean, the balancing settings are related to the item, aren't they?

Reply:
But not only to the item and it's rather unintuitive to look for settings like armor at begin or general classic/new armor system in the armor item. Other items might also use the armor system

Reply:
Armor at begin isn't located in this menu for this very reason. If other items also use the armor system, what does it change?

Reply:
If this is not included then why include settings that are related to the armor system in general on a specific item instead of including it in gameplay settings where a user expects them?

Reply:
Because "armor on spawn" is a gameplay setting. The handling of the armor itself is a item setting. At least in my opinion the user expects armor settings within the armor item. Especially since crowbar, DNA scanner, teleporter, ... settings are all located in their items instead of the gameplay menu. Why should the armor be different?

Reply:
Because armor is a much more general system than the dna scanner etc, thats just my opinion. 
Any one else want to comment on this? @Histalek or @ZenBre4ker ?

Reply:
I dont know much about armor, but I agree that placing it in the item-system is really unintuitive, Id prefer general armor settings in a gameplay menu. And only stuff that classically relates to a buyable item should be here. Its weird, that armor is an item anyway. For me that would be two different things. An item I can buy to get armor in the first place and a general system that decides who gets the armor at the start of a round depending on roles.

Reply:
Okay, Svens wording let me understand Tobis point. I'm pretty sure he meant the same thing. But there are two things: An armor system and an armor item.
I will add a new submenu called somthing like "Player Settings" where I add this stuff to. There are a handful more settings that could be moved there

Reply:
done.